ReactNativeClient/lib/components/screens/config.js

```diff
@@ -217,6 +217,10 @@ class ConfigScreenComponent extends BaseScreenComponent {
 				color: theme.color,
 				flex: 1,
 			},
+			exportPathField: {
+				marginRight: 20,
+				color: theme.color,
+			},
```

Comment:
Please could you create a more generic style that would be used by all config TextInput fields?

Reply:
> Please could you create a more generic style that would be used by all config TextInput fields?

Done Sir, Please have a look.